auth-web/src/components/auth/manage-business/manage-business-dialog/ManageBusinessDialog.vue

```diff
@@ -302,6 +335,15 @@ export default defineComponent({
       return store.dispatch('business/updateBusinessName', businessNumber)
     }
 
+    const invitationToAccount = ref({ 'name': null, 'uuid': null })
+    const invitationAdditionalMessage = ref('')
+    const requestAuthRegistryFormIsValid = ref(false)
```

Comment:
Should probably use reactive for these, but it's fine we can refactor it later

Reply:
https://github.com/bcgov/sbc-auth/blob/8842e93419e2fb79a238428328ebdb2563652252/auth-web/src/stores/staff.ts#L292